src/index.ts

```diff
@@ -1,8 +1,9 @@
-import $rdf, { NamedNode } from 'rdflib'
+import * as $rdf from 'rdflib'
```

Comment:
It would indeed be clearer if this would be `import * as $rdf, { NamedNode } from 'rdflib'` I think.